src/features/customerView/billModal/BillModal.tsx

```diff
@@ -26,7 +26,7 @@ const BillModal = ({ bill, toggleModal, ...modalProps }: BillModalProps) => {
 
   const { contractPeriod } = bill;
   return (
-    <Modal isOpen={true} {...modalProps}>
+    <Modal isOpen toggleModal={() => toggleModal?.(false)} {...modalProps}>
```

Comment:
Nit: Isn't specifying `isOpen` here superfluous, as we override it with `{...modalProps}`?

```suggestion
    <Modal toggleModal={() => toggleModal?.(false)} {...modalProps}>
```

Reply:
`isOpen` actually was not in `{...modalProps}` as it was omitted in the type and not in the props. I decided to add the prop and make it like your suggestion for consistency. Also replaced `isOpen={true}` with `isOpen` for all Modals